### PR TITLE
Add Release.MD

### PR DESCRIPTION
**Pull request by zhenlu**

### CODE REVIEW

**Comment by zhenlu** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #44** <a href="https://app.graphite.dev/github/pr/lightsparkdev/lightspark-crypto-uniffi/44?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/lightsparkdev/lightspark-crypto-uniffi/44?utm_source=stack-comment).

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Step 2: Release
+
+Create a release tag and make a release on Github.
+
+## Step 3: Publish to Go
+
+Run the following command to publish the go library.
+
+```bash
+$ GOPROXY=proxy.golang.org go list -m github.com/lightsparkdev/lightspark-crypto-uniffi/lightspark-crypto-go@[version]
```
Comment (jklein24) [inline RELEASE.md:36]: Fwiw, I actually don't think this command is necessary every time, it just forces the cache to be cleared quicker. @mhrheaume would know better than I do though.

**Review by jklein24 — APPROVED** [review]

**Comment by zhenlu** [comment]
### Merge activity

* **Dec 15, 4:47 PM**: [Graphite](https://app.graphite.dev/github/pr/lightsparkdev/lightspark-crypto-uniffi/44) rebased this pull request as part of a merge.

File: RELEASE.md (hunk 1 above)
Comment (mhrheaume) [reply RELEASE.md:36]: I thought that too but the official docs don't really make it super clear: https://go.dev/doc/modules/publishing . Guess it doesn't hurt, though.

Comment (zhenlu) [reply RELEASE.md:36]: I actually encountered an error this time when I tried go get before I did go list. It says the version cannot be found.

**Pull request merged**